plugins/source/azure/CONTRIBUTING.md

```diff
@@ -1,53 +1,114 @@
-## CloudQuery Azure Source Plugin Contributing Guidelines
+# Azure Source Plugin Contribution Guide
 
-## Adding a new resource
+Thanks for contributing to CloudQuery! You are awesome. This document serves as a guide for adding new services and resources to the Azure source plugin.
 
-Some information can be found in the [docs for developing a new provider](https://docs.cloudquery.io/developers/developing-new-provider).
+There are two steps to adding a new Azure resource:
 
-As a prerequisite, ensure that API calls to list/describe the desired resource are present in
-[github.com/Azure/go-autorest/autorest](https://pkg.go.dev/github.com/Azure/go-autorest/autorest), and make note of:
+1. [Generate interfaces for the Azure SDK function(s) that fetch the resource](#1-generate-interfaces-for-the-azure-sdk-functions-that-fetch-the-resource)
+2. [Add a code generation recipe](#2-add-a-code-generation-recipe)
+
+As a prerequisite, ensure that API calls to list/describe the desired resource are present in [the old Azure client](https://github.com/Azure/azure-sdk-for-go#client-previous-versions) or [the new one](https://github.com/Azure/azure-sdk-for-go#management-new-releases), and make note of:
 
 - to which Azure service the resource belongs
 - the schema of the returned object(s)
 
-## Setting up the service
+## 1. Generate Interfaces for the Azure SDK Function(s) that Fetch the Resource
+
+### Before you Start
+
+Inside the root of this repository, run:
+
+```shell
+make install-tools
```

Comment:
I guess this should be updated now that the root makefile has been removed? Or shall we bring it back, at least just for this command?

Reply:
Ah sorry, I see, the command is in the Azure makefile. Maybe we should just rephrase the above to say `in the azure directory` or something like that